src/main/java/hudson/plugins/svn_tag/SvnTagPublisher.java

```diff
@@ -43,11 +45,14 @@
 
     private String tagDeleteComment = null;
 
+	private Integer waitBeforeTagging = 0;
+
     @DataBoundConstructor
-    public SvnTagPublisher(String tagBaseURL, String tagComment, String tagDeleteComment) {
+    public SvnTagPublisher(String tagBaseURL, String tagComment, String tagDeleteComment, Integer waitBeforeTagging) {

```

Comment:
Causes a binary compatibility issue, because it is not marked as "no external use". The original constructor should be retained.